### PR TITLE
TDR-3157 Judgement before you upload content changes

### PR DESCRIPTION
**Pull request by thanhz**

### CODE REVIEW

File: app/views/judgment/judgmentBeforeUploading.scala.html [hunk 1]
```diff
@@ -3,32 +3,36 @@
...
+                <p class="govuk-body">Your upload must contain the following information:</p>
+                <ul class="govuk-list">
+                    <li>neutral citation</li>
+                    <li>name(s) of judge(s)</li>
+                    <li>name(s) of parties</li>
+                    <li>court and judgment date</li>
+                </ul>
+                <h2 class="govuk-heading-m">Do you need to contact us about this transfer?</h2>
+                <p class="govuk-body">
+                    Send an email to <a class="govuk-link" href="mailto:@Messages("nationalArchives.judgmentsEmail")?subject=Ref: @consignmentRef">judgments@@nationalarchives.gov.uk</a>
```
Comment (vimleshtna) [inline app/views/judgment/judgmentBeforeUploading.scala.html:21]: You can use the `@Messages("nationalArchives.judgmentsEmail")` for the link text as well:

`<a href="mailto:@Messages("nationalArchives.judgmentsEmail")?subject=Ref: @consignmentRef">@Messages("nationalArchives.judgmentsEmail")</a>`

File: app/views/judgment/judgmentBeforeUploading.scala.html [hunk 2]
```diff
@@ -3,32 +3,36 @@
...
-    </div>
-    @transferReference(consignmentRef, isJudgmentUser = true)
-</div>
-}
-}
+                <div class="govuk-button-group">
+                    <a href="@routes.UploadController.judgmentUploadPage(consignmentId)" role="button" draggable="false" class="govuk-button" data-module="govuk-button">Continue</a>
+                    <a class="govuk-link" href="@routes.HomepageController.homepage()">Cancel</a>
+                </div>
+                </div>
```
Comment (vimleshtna) [inline app/views/judgment/judgmentBeforeUploading.scala.html:34]: Can you please fix the indentation of the closing </div>?

**Review by vimleshtna — COMMENTED** [review]
Looks good, just a few minor comments. Thanks

File: app/views/judgment/judgmentBeforeUploading.scala.html (hunk 2 above)
Comment (thanhz) [reply app/views/judgment/judgmentBeforeUploading.scala.html]: should be fixed now, intellij auto indent is quite unreliable :anger:

**Pull request merged**